parser.y

```diff
@@ -1187,6 +1187,7 @@ import (
 	PlacementCountOpt                      "Placement rules count option"
 	PlacementLabelOpt                      "Placement rules label option"
 	PlacementRoleOpt                       "Placement rules role option"
```

Comment:
The 'opt' means 'optional'... not table 'option'.

Please check other rules that end with 'Opt': `AnalyzeOptionListOpt`, `AlterTableSpecListOpt`, etc.

Could you file another PR to correct these names?

Reply:
OTOH we've got things like `CreateViewSelectOpt` and `IndexNameAndTypeOpt`...